std/process.d

```diff
@@ -570,7 +559,7 @@ private Pid spawnProcessImpl(in char[][] args,
             string errorMsg;
             readExecResult = read(forkPipe[0], &error, error.sizeof);
 
-            switch(status)
+            switch (status)
```

Comment:
Yes. If almost all enum members are present in the `switch` and it makes sense to account for them explicitly, then use `final switch` with an `assert(false)` case body for cases which logically cannot be expected to occur for that `switch` expression in that point in the program.